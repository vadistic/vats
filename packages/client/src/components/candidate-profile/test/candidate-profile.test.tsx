import React from 'react'
// tslint:disable-next-line:no-implicit-dependencies
import { render } from 'react-testing-library'

import { FixtureWrapper } from '../../fixture/wrapper'
import { CandidateProfileFixture } from './fixture'

it('renders without crashing', () => {
  const div = document.createElement('div')
  const { asFragment } = render(
    <FixtureWrapper>
      <CandidateProfileFixture />
    </FixtureWrapper>,
  )

  expect(asFragment()).toMatchSnapshot()
})
