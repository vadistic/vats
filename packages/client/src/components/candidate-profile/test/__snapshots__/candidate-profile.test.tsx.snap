// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="ms-Fabric fabric root-47"
  >
    <p>
      TODO: Candidate profile
    </p>
  </div>
</DocumentFragment>
`;
